src/ImageSharp/Formats/Jpeg/Components/Decoder/JFifMarker.cs

```diff
@@ -73,7 +73,8 @@ public static bool TryParse(ReadOnlySpan<byte> bytes, out JFifMarker marker)
     {
         // Some images incorrectly use JFXX as the App0 marker (Issue 2478)
         if (ProfileResolver.IsProfile(bytes, ProfileResolver.JFifMarker)
-            || ProfileResolver.IsProfile(bytes, ProfileResolver.JFxxMarker))
+            || ProfileResolver.IsProfile(bytes, ProfileResolver.JFxxMarker)
+            || ProfileResolver.IsProfile(bytes, ProfileResolver.OSQidMaker))
```

Comment:
While this works, I'm not sure it's the correct fix. 

I think we might be able to simply remove the throw when the marker doesn't match a JFIF one and skip the remaining bytes. 

What is OSQidMaker? Can you please supply more information?

Reply:
In fact, in my research I didn't find markers that specified such a conversion starting with these bytes, but I have images of a camera that always has these bytes at the beginning. I also had success in all tests when ignoring this if but I don't know what more problems this could cause. I'm running a fork of the lib in my application and I don't get any more errors

Reply:
The JFIF JFXX headers contain metadata and we can actually operate without them. Your images actually contain a JFIF header in an APP0 block just after the camera junk. 

I can’t accept the fix as-is because the values are non-standard so we should try a more robust solution as suggested